server/block/explosion.go

```diff
@@ -132,6 +132,14 @@ func (c ExplosionConfig) Explode(tx *world.Tx, explosionPos mgl64.Vec3) {
 			}
 		}
 	}
+
+	ctx := event.C(tx)
+	spawnFire := c.SpawnFire
+	itemDropChance := c.ItemDropChance
+	if tx.World().Handler().HandleExplosion(ctx, &affectedEntities, &affectedBlocks, &itemDropChance, &spawnFire); ctx.Cancelled() {
```

Comment:
The explosion position would also be nice to have. For example, you might want to contain an explosion's effects within a plot which would require you to be able to identify the plot the explosion started from.